test/gst-vaapi/encode/vp9.py

```diff
@@ -65,17 +63,101 @@ def test(self, case, gop, bitrate, fps, refmode, looplvl, loopshp):
       minrate   = bitrate,
       rcmode    = "cbr",
       refmode   = refmode,
+      lowpower  = False,
     )
-    self.encode()
+  @slash.requires(*platform.have_caps("encode", "vp9_8"))
+  @slash.requires(*have_gst_element("vaapivp9enc"))
+  @slash.requires(*have_gst_element("vaapivp9dec"))
+  @slash.parametrize(*gen_vp9_cbr_parameters(spec))
+  def test(self, case, gop, bitrate, fps, refmode, looplvl, loopshp):
+      self.init(spec, case, gop, bitrate, fps, refmode, looplvl, loopshp)
+      self.encode()
 
 class vbr(VP9EncoderTest):
+  def init(self, spec, case, gop, bitrate, fps, refmode, quality, looplvl, loopshp):
```

Comment:
The command `LIBVA_DRIVER_NAME=i965 ./vaapi-fits run test/gst-vaapi/encode/vp9.py --parallel-metrics --platform KBL -v -k vbr` generates error `NameError: global name 'tspec' is not defined`.

`spec` parameter name should be `tspec`